dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -330,6 +333,53 @@ def kurtosis(self):
                     self.match_count,
                     self._biased_kurtosis)
 
+    @staticmethod
+    def _perform_t_test(mean1, var1, n1,
+                        mean2, var2, n2):
+        results = {}
+        s_delta = var1/n1 + var2/n2
+        t = (mean1 - mean2) / np.sqrt(s_delta)
+        conservative_df = min(n1, n2) - 1
+        welch_df = s_delta ** 2 / ((var1 / n1) ** 2 / (n1 - 1) + (var2 / n2) ** 2 / (n2 - 1))
+        results.update({
+            "t-statistic": t,
+            "conservative_df": conservative_df,
```

Comment:
is conservative the term? vs worst-case?

Reply:
my thoughts are worst-case is clear in its definition whereas conservative has varying degrees

Reply:
I haven't heard of this method called "worst-case" before. All source I've seen refer to this as the conservative estimate for the degrees of freedom.